modules/sc-mesh-secure-deployment/src/nats/mdm_agent.py

```diff
@@ -80,6 +79,8 @@ def __init__(
             self.logger,
             self.__lock
         )
+        self.__url: str = "defaultmdm.local:5000"  # mDNS callback updates this one
+        self.__fallback_url: str = f"[{self.cbma_ctrl.__IPV6_WHITE_PREFIX}::1]:5000"
```

Comment:
This line won't work.

Reply:
Can you try this? https://github.com/tiiuae/mesh_com/pull/450#discussion_r1587258156

Reply:
it's still private.

Reply:
It should be self.__fallback_url: str = f"[{self.cbma_ctrl._CBMAAdaptation__IPV6_WHITE_PREFIX}::1]:5000" in order to work. But better way would be probably move those constant definitions to constants.py instead. Perhaps also some of the other magic prefixes from 99_at_boot could be given some purpose via naming some of those as well.

Reply:
Fixed in https://github.com/tiiuae/mesh_com/pull/450/commits/f09732dc32dcdab1e314eaee41e5af4f41e5f59f :1st_place_medal: